Gemfile

```diff
@@ -9,11 +9,10 @@ gem "jwt", "~> 2.2"
 gem "sass-rails", "~> 5.0"
 gem "slimmer", "~> 13.0"
 gem "uglifier", "~> 4.1"
-
 gem "gds-api-adapters", "~> 60.1"
 gem "govuk_app_config", "~> 2.0"
-gem "govuk_publishing_components", "~> 17.21"
-gem "plek", "~> 3.0"
+gem 'govuk_publishing_components', '~> 21.0.0'
+gem 'plek', '~> 3.0'
```

Comment:
Sorry to nitpick, but why have you changed the `"` to `'`? It's inconsistent with the rest of the file. Also this is one reason why the build is failing, the linter prefers double quotes.

Reply:
Fixed the quotes now.